### ISSUE TITLE
Skip Reactome reactions that contain the word 'dissociates' in the label

### ISSUE THREAD

**Issue by ukemi** [geneontology/pathways2GO#198]
If I am not mistaken, GO does not consider dissociation a valid MF. As I work through various signaling models, I think it is safe to create a rule where the importer skips a node if the Reactome reaction contains the word 'dissociates'. Since there is no intervening MF in these cases, The relations between the dissociation node should be calculaed as follows in accordance with the new MF to MF rules.

1. If reaction(1) directly positively regulates reaction(dissociation) directly positively regulates reaction(3)-> reaction(1) directly positively regulates reaction(3).
2. If reaction(1) directly negatively regulates reaction(dissociation) directly negatively regulates reaction(3)-> reaction(1) directly negatively regulates reaction(3).
3. If reaction(1) causally related to reaction(dissociation) has output (x) and reaction(3) has input (x)-> reaction(1) directly provides input for reaction(3).
4. If reaction(1) directly provides input for reaction(dissociation) directly positively regulates reaction(3)-> reaction(1) directly positively regulates reaction(3).
5. All other cases follow existing property chains.

**Comment by deustp01** [comment]
But preserve the original plan at the top of the ticket, maybe for a review in October at NYU, to look for places where there may be enablers hidden inside events where the nominal action is that an input comes apart to yield outputs that were the input's parts.

**Comment by ukemi** [comment]
Yes, for those we decide to skip, I think the plan above is a good  start. I will add this ticket to the agenda for Friday.

**Comment by deustp01** [comment]
And if we’re making good progress with the binding events in our pathways earlier in the week, we can sneak a look at dissociation ones that are there, at least to classify them according to the scheme in the ticket or to refine that scheme.

From: David Hill ***@***.***>
Reply-To: geneontology/pathways2GO ***@***.***>
Date: Tuesday, September 26, 2023 at 10:07 AM
To: geneontology/pathways2GO ***@***.***>
Cc: "D'eustachio, Peter" ***@***.***>, Assign ***@***.***>
Subject: Re: [geneontology/pathways2GO] Skip Reactome reactions that contain the word 'dissociates' in the label (Issue #198)


[EXTERNAL]

Yes, for those we decide to skip, I think the plan above is a good start. I will add this ticket to the agenda for Friday.

**Comment by deustp01** [comment]
And if we’re making good progress with the binding events in our pathways earlier in the week, we can sneak a look at dissociation ones that are there, at least to classify them according to the scheme in the ticket or to refine that scheme.
